isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/detailkonzept-komponente-service/inhalt.adoc

```diff
@@ -14,23 +14,29 @@ Diese technische Komponente besteht aus zwei Teilen: dem Service-Framework und d
 
 Das *Service-Framework* bezeichnet die konkrete Technologie, mit der die Services des Anwendungskerns zur Verfügung gestellt werden.
 
-Die *Servicelogik* enthält Daten und Funktionalität, die für die Bereitstellung des Services relevant sind.
-
+Die Fachkomponenten der *Servicelogik* Servicelogik beinhalten Service-Endpoints, 
```

Comment:
Was sind Service-Endpoints? Wir sollten beim bestehenden Sprachgebrauch bleiben und keine neuen Begriffe einführen. Zudem sollten nur die Passagen angepasst werden, die durch die Ticketanforderungen betroffen sind. Dies sehe ich hier nicht. Ich würde daher die hinzugefügten Zeilen 17-20 entfernen. Die Inhalte der Service-Logik werden im weiteren Dokument noch beschrieben.

Reply:
Service Endpoints sind Klassen, die von Service-Framework aufgerufen werden, um Requests zu verarbeiten.
Hier wurde ein neuer (technologie-unabhängiger) Begriff gesucht um den Aufbau der Service-Komponente zu beschreiben 
* Service-Logik der Service-Komponente beinhaltet Fachkomponenten, 
* Fachkomponenten beinhalten Service-Endpoints, 
* Methoden der Service-Endpoints rufen Methoden von Klassen von Fachkomponenten des Anwendungskerns

Reply:
Ich verstehe noch nicht, warum die bisherige Beschreibung ohne den Begriff "Service-Endpoints" nicht technologie-unspezififisch sein soll? Im Kapitel "Service-Logik" steht nun 
"Die Servicelogik gliedert sich, wie der Anwendungskern, in *Fachkomponenten*.
Die Fachkomponenten setzen im wesentlichen *Service-Schnittstellen* um."

Service-Endpoints und Service-Schnittstellen sind für mich redundant. Daher würde ich nur bei Service-Schnittstellen bleiben und diesen Absatz überhaupt nicht anfassen. Ich sehe hier keinen Bezug zu einer bestimmten Technologie, sodass dieser Absatz überhaupt angepasst werden muss.

Reply:
Als REST spezifische Ausprägung der Service-Endpoints haben wir die Controller Klassen, letztere assoziiert jeder mit der REST Technologie, daher wollte ich diese Bezeichnung vermeiden

Ersetzung des Begriffs Service-Endpoints durch Service-Schnittstellen führt zu Verwirrungen denn
einerseits "Die Service-Schnittstellen bestehen aus einer Service-Fassade und einer Exception-Fassade"
andererseits "Service-Fassade besteht aus Fachkomponenten mit Service-Schnittstellen"

womit sich nach Ersetzung des Begriffs  Service-Endpoint eine zyklische Definition ergibt ...